### PR TITLE
issue 690 fixing dragging examples

### PR DESCRIPTION
**Pull request by nmccready**

### CODE REVIEW

File: examples/0500-markers-simple-example.html [hunk 1]
```diff
@@ -30,8 +30,11 @@
                     lat: 51,
                     lng: 0
                 },
```
Comment (nmccready) [inline examples/0500-markers-simple-example.html]: fixing example

File: examples/0504-markers-popup-example.html [hunk 2]
```diff
@@ -26,7 +26,10 @@
                     }
                 },
                 events: {
-                    markers: [ 'dragend' ]
+                    markers: {
+                      enable: [ 'dragend' ]
```
Comment (nmccready) [inline examples/0504-markers-popup-example.html]: fixing example

**Comment by nmccready** [comment]
@tombatossals @jessertaylor merging on travis passing; since this is just example fixes

**Comment by tombatossals** [comment]
LGTM :+1

**Pull request merged**